cmd/metagenomics/shortReadMNGS.go

```diff
@@ -0,0 +1,106 @@
+package metagenomics
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/chanzuckerberg/czid-cli/pkg/util"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var projectName string
+var stringMetadata map[string]string
+var metadataCSVPath string
+var disableBuffer bool
+var technology string
+var guppyBasecallerSetting string
+var workflow string = "short-read-mngs"
```

Comment:
Can this get set with "long-read-mngs" ?

Reply:
it gets set with an if below